usr/share/rear/rescue/default/500_ssh.sh

```diff
@@ -24,10 +24,14 @@ if has_binary sshd; then
     Log "Adding required libfreeblpriv3.so to LIBS"
 
     # copy ssh user
-    if PASSWD_SSH=$(grep ssh /etc/passwd) ; then
+    PASSWD_SSH=$(getent passwd ssh sshd)
+    if test -n "$PASSWD_SSH" ; then
     # sshd:x:71:65:SSH daemon:/var/lib/sshd:/bin/false
-        echo "$PASSWD_SSH" >>$ROOTFS_DIR/etc/passwd
         IFS=: read user ex uid gid gecos homedir junk <<<"$PASSWD_SSH"
+	# skip if this user exists already in the restore system
+	if ! egrep -q "^$user:" $TARGET_FS_ROOT/etc/passwd ; then
```

Comment:
Why not add it to `CLONE_USERS` similar to the ssh group?

Reply:
I think the purpose of this 500_ssh.sh script should be kept
separated from the CLONE_USERS functionality
what I call "KSIS" Keep Separated Issues Separarated
(cf. RFC 1925 item 5 ;-)